File: tests/testSrc/com/intellij/lang/jsgraphql/frameworks/apollo/GraphQLApolloValidationTest.java
package com.intellij.lang.jsgraphql.frameworks.apollo;

import com.intellij.lang.jsgraphql.GraphQLTestCaseBase;
import org.jetbrains.annotations.NotNull;

public class GraphQLApolloValidationTest extends GraphQLTestCaseBase {
  @Override
  protected @NotNull String getBasePath() {
    return "/frameworks/apollo/validation";
  }

  public void testLocalFields() {
    doHighlightingTest();
  }
}

File: tests/testSrc/com/intellij/lang/jsgraphql/frameworks/apollo/GraphQLApolloValidationTest.kt
package com.intellij.lang.jsgraphql.frameworks.apollo

import com.intellij.lang.jsgraphql.GraphQLTestCaseBase
import com.intellij.openapi.progress.runBlockingCancellable

class GraphQLApolloValidationTest : GraphQLTestCaseBase() {
  override fun getBasePath(): String {
    return "/frameworks/apollo/validation"
  }

  fun testLocalFields() = runBlockingCancellable {
    doHighlightingTest()
  }
}
